Tasks/DownloadPackageV0/download.ts

```diff
@@ -165,7 +166,7 @@ export async function unzip(zipLocation: string, unzipLocation: string): Promise
 
 	await new Promise<void>(function (resolve, reject) {
 		if (tl.exist(unzipLocation)) {
-			tl.rmRF(unzipLocation, false);
+			tl.rmRF(unzipLocation);
```

Comment:
why is this change reqd.?

Reply:
I updated the vsts-task-lib version, which requires this change. Different versions of vsts-task-lib versions in task and utils was leading to issues.